pkg/asset/installconfig/vsphere/folder_altinfra.go

```diff
@@ -0,0 +1,44 @@
+//go:build altinfra
+// +build altinfra
```

Comment:
What happens when capi is promoted to the main container image and the `altinfra` is not used anymore?

Reply:
I mention that in the commit message, the function in altinfra is put back into `vsphere.go`